### ISSUE TITLE
Adding query parameters to loadFlutterAsset when loading a local html file

### ISSUE THREAD

**Issue by Wian-TMC** [flutter/flutter#97370]
## Use case

Being able to pass query parameters can make local asset files more dynamic without having to duplicate asset files fir minor changes (example would be having different logic for different environments)

## Proposal

Adding a secondary parameter to the loadFlutterAsset function which can accept a Map<String, String> for query parameters when loading local html files.

**Comment by goderbauer** [comment]
Assuming you're taking about https://pub.dev/documentation/webview_flutter/latest/webview_flutter/WebViewController/loadFlutterAsset.html? Since that's the webview plugin, I am going to remove the framework label for this.

**Comment by stuartmorgan** [comment]
Is this functionality actually supported by the system webviews for file-based loads?

**Comment by timukasr** [comment]
> Is this functionality actually supported by the system webviews for file-based loads?

Yes, for example this is works as expected on Android `webView.loadUrl("file:///android_asset/www/index.html?locale=en");`.

**Comment by ahsanrahman0745** [comment]
any progress on this?

**Comment by timukasr** [comment]
@ahsanrahman0745 I meant that using native Android WebView, then query parameters works. In flutter (your example) does not work.
